src/components/ProfileCard/NetworkOption/NetworkOption.tsx

```diff
@@ -99,12 +128,43 @@ export const NetworkOption: React.FC = () => {
         open={Boolean(anchorEl)}
         onClose={handleClose}>
         {networkOptions.map(option => (
-          <MenuItem key={option.id} onClick={() => handleSelected(option.id)}>
+          <MenuItem
+            key={option.id}
+            onClick={() => handleSelected(option.id)}
+            className={option.id === current ? styles.menu : ''}>
             <ListItemIcon>{icons[option.id as keyof typeof icons]}</ListItemIcon>
             <ListItemText>{option.title}</ListItemText>
           </MenuItem>
         ))}
       </Menu>
+      <PromptComponent
```

Comment:
try to use the new implemented prompt hooks like in this PR https://github.com/myriadsocial/myriad-web/pull/1035